File: bronze/11005/python/main.py

if (__name__ == "__main__"):
    N, B = map(int, input().split())
    digit = 0
    result = ""
    while (N > 0):
        digit = int(N % B)
        N = int(N / B)
        if (digit >= 10):
            result = chr(65 + digit - 10) + result
        else :
            result = chr(48 + digit) + result
    print(result)

File: bronze/11005/c++/main.cpp
#include <iostream>
#include <string>

int main()
{
    int N, B;
    std::cin >> N >> B;
    std::string result;

    int digit = 0;
    
    while (N > 0) {
        digit = N % B;
        N = N / B;

        if (digit >= 10) {
            result = static_cast<char>(65 + digit -10) + result;
        } else {
            result = static_cast<char>(48 + digit) + result;
        }
    }
    std::cout << result;
    

    return 0;
}